docs/core/tutorials/netcore-hosting.md

```diff
@@ -89,7 +89,7 @@ Unlike the mscoree.h hosting API (described below), CoreCLRHost.h APIs start the
 
 ### Step 5 - Run managed code!
 
-With the runtime started, the host can call managed code. This can be done in a couple of different ways. The sample code linked to this tutorial uses the `coreclr_create_delegate` function to create a delegate to a static managed method. This API takes the assembly name, namespace-qualified type name, and method name as inputs and returns a delegate that can be used to invoke the method.
+With the runtime started, the host can call managed code. This can be done in a couple of different ways. The sample code linked to this tutorial uses the `coreclr_create_delegate` function to create a delegate to a static managed method. This API takes the [assembly name](https://docs.microsoft.com/dotnet/framework/app-domains/assembly-names), namespace-qualified type name, and method name as inputs and returns a delegate that can be used to invoke the method.
```

Comment:
This and the link on line 191 should be relative links rather than an URL: ../../framework/app-domains/assembly-names.md